nemo/collections/llm/recipes/llama3_70b.py

```diff
@@ -155,6 +156,7 @@ def pretrain_recipe(
         name (str): Name of the pre-training run.
         num_nodes (int): Number of compute nodes to use.
         num_gpus_per_node (int): Number of GPUs per node.
+        performance_mode (bool): If true, enables optimizations for maximum performance.
```

Comment:
@marcromeyn does this look good to you? Context- your comment here- https://github.com/NVIDIA/NeMo/pull/10883#discussion_r1803220645